<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WholeTrackDlg</class>
 <widget class="QDialog" name="WholeTrackDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>媒体库</string>
  </property>
  <widget class="CButton" name="btnReturn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/btnReturn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="CLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>48</y>
     <width>420</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="CButton" name="btnPlay">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>40</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/btnPlay.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="CLabel" name="labelDisk">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CButton</class>
   <extends>QPushButton</extends>
   <header location="global">widget/button.h</header>
  </customwidget>
  <customwidget>
   <class>CLabel</class>
   <extends>QLabel</extends>
   <header location="global">widget/label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
